cob_hardware_config/cob4-5/config/head_self_collision.yaml

```diff
@@ -0,0 +1 @@
+
```

Comment:
the cob4-5 head will be upgraded later on, so let's keep the config files for the head